src/Microsoft.Azure.WebJobs.Logging/Entities/ContainerActiveEntity.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.WindowsAzure.Storage.Table;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    // Entity to track active compute Containers during a timerange.
+    // A Container is a billable compute unit, like a virtual machine. 
+    // A container may run many functions simultaneously. 
+    public class ContainerActiveEntity : TableEntity
+    {
+        const string PartitionKeyFormat = TableScheme.ContainerActivePK;
```

Comment:
This is tricky. I'll followup in email. 

---
In reply to: [56055211](https://github.com/Azure/azure-webjobs-sdk/pull/655#discussion_r56055211) [](ancestors = 56055211)